export.go

```diff
@@ -214,11 +247,23 @@ func (conn *Conn) Emit(path ObjectPath, name string, values ...interface{}) erro
 //
 // Export returns an error if path is not a valid path name.
 func (conn *Conn) Export(v interface{}, path ObjectPath, iface string) error {
+	return conn.ExportWithMap(v, nil, path, iface)
+}
+
+// ExportWithMap works exactly like Export but provides the ability to remap
+// method names (e.g. export a lower-case method).
+//
+// The keys in the map are the real method names (exported on the struct), and
+// the values are the method names to be exported on DBus.
```

Comment:
I could have gone either way on the mapping direction (i.e. keys = actual, values = dbus or keys = dbus, values = actual), but settled on the direction that seemed to make the most sense from the user's aspect: exporting left to right, or key to value.